testsuite/pom.xml

```diff
@@ -28,6 +28,7 @@
   <artifactId>nxrm3-maven-plugin-testsuite</artifactId>
   
   <properties>
+    <nexus3.version>3.41.1-01</nexus3.version>
```

Comment:
Haven't we released 3.42? Why 3.41.1?

What will ensure we update this upon releases?

Reply:
Checking RSO , seems like there's no 3.42.0 for the nxrm-pro-image package : 
![Screen Shot 2022-10-25 at 4 20 29 PM](https://user-images.githubusercontent.com/36464079/197884279-2685080f-60ed-431b-b2c9-e3f729809013.png)